core/extensions.js

```diff
@@ -17,7 +17,8 @@
  * @name Blockly.Extensions
  * @namespace
  */
-goog.provide('Blockly.Extensions');
+goog.module('Blockly.Extensions');
```

Comment:
Since this doesn't export a class it should (eventually) be renamed `Blockly.extensions`.  I've added it to the list in #5073.